content/docs/manuals/networking-internals/mqtt-authentication.md

```diff
@@ -0,0 +1,163 @@
+    ```
+2. Generate the certificates in the `./certs` directory
+    * **MQTTS (Server):**
+       1. Generate CA authority key:
+          ```bash
+          openssl req -new -x509 -days <duration> -extensions v3_ca -keyout ca.key -out ca.crt
+          ```
+       2. Generate a server key:
+          ```bash
+          openssl genrsa -out server.key 2048
+          ```
+       3. Generate a certificate signing request including the URL as a SAN:
+          ```bash
+          openssl req -out server.csr -key server.key -new -addext "subjectAltName = IP:${SYSTEM_MANAGER_URL}, DNS:mqtts"
+          ```
+            When prompted for the CN, enter `mqtts`
+       4. Send the CSR to the CA:
+           ```bash
+           openssl x509 -req -in server.csr -CA ca.crt -CAkey ca.key -CAcreateserial -out server.crt -days <duration> -copy_extensions copyall
+           ```
+       5. Grant permissions to read the server keyfile:
+            ```bash 
+            chmod 0644 server.key
+            ```
+    * **Cluster Manager (Client):**
+        1. Generate a client key:
+            ```bash
+            openssl genrsa -aes256 -out cluster.key 2048
+            ```
+        2. Generate a certificate signing request:
+            ```bash
+            openssl req -out cluster.csr -key cluster.key -new
+            ```
+            When prompted for the CN, enter `cluster_manager`
+        3. Send the CSR to the CA:
+            ```bash
+            openssl x509 -req -in cluster.csr -CA ca.crt -CAkey ca.key -CAcreateserial -out cluster.crt -days <duration>
+            ```
+        4. Export the keyfile password as an environment variable:
+            ```bash
+            export CLUSTER_KEYFILE_PASSWORD=<keyfile password>
+            ```
+    * **Cluster Service Manager (Client):**
+        1. Generate a client key:
+            ```bash
+            openssl genrsa -aes256 -out cluster_net.key 2048
+            ```
+        2. Generate a certificate signing request:
+            ```bash
+            openssl req -out cluster_net.csr -key cluster_net.key -new
+            ```
+            When prompted for the CN, enter `cluster_service_manager`
+        3. Send the CSR to the CA:
+            ```bash
+            openssl x509 -req -in cluster_net.csr -CA ca.crt -CAkey ca.key -CAcreateserial -out cluster_net.crt -days <duration>
+            ```
+        4. Export the keyfile password as an environment variable:
+            ```bash
+            export CLUSTER_SERVICE_KEYFILE_PASSWORD=<keyfile password>
+            ```
+
+3. Deploy the cluster with the MQTTS override
+    ```bash
+    sudo -E docker compose -f docker-compose.yml -f override-mosquitto-auth.yml
+    ```
+
+### Configuring a Node
+<!--- Subject to change when NodeEngine and NetManager are demonized -->
+
+1. Copy the `ca.crt` and `ca.key` files to the worker node.
+2. Generate the certificates
+   1. Generate a client key:
+       ```bashopenssl genrsa -aes256 -out client.key 2048
```

Comment:
```suggestion
       ```bash
       openssl genrsa -aes256 -out client.key 2048
```

Otherwise :

![image](https://github.com/user-attachments/assets/4d03db59-c6ec-4e56-aa3f-7377f73dd5c8)